runtime/lua/vim/snippet.lua

```diff
@@ -674,7 +674,9 @@ end
 --- @return boolean
 function M.active(filter)
   local active = M._session ~= nil and M._session.bufnr == vim.api.nvim_get_current_buf()
-
+  if M._session.current_tabstop and M._session.current_tabstop.index == 0 then
+    return active and false
```

Comment:
`active and false` will always return falsy. 
```suggestion
    return false
```